### ISSUE TITLE
Division by zero when no days left

### ISSUE THREAD

**Issue by marier-nico** [marceloFA/deadlines_app#7]
Something I noticed while playing around with the app is that if you create a task that has a deadline tomorrow, and click it to view details, the app will crash with a divide by zero error. This part of the stack track seems relevant. I'm guessing it crashes because total days is an int, and if you have less than one day to do a task, `total_days` will be less than 1, which will be 0 if you do `int()` of it somewhere.

```
File "deadlines_app/tasks/views.py" in get_progress_percentage
  100.     progress_percentage = 100 - (100 * task.days_left / total_days)

Exception Type: ZeroDivisionError at /tasks/3
Exception Value: division by zero
```

**Comment by marceloFA** [comment]
Ver well noticed, thanks for that! Would you like to work on a fix for this issue, @marier-nico?  Otherwise I'll handle this

**Comment by BusinessFawn** [comment]
I can do it :)

**Comment by marceloFA** [comment]
Thanks for helping out!

**Comment by marceloFA** [comment]
Issue resolved, #8 fixed this.
